src/config/dynamic-theme-fixes.config

```
@@ -44,6 +44,15 @@ div[class^='scrollableList'] {
 
 ================================
 
+*.eduserver.ru
```

```suggestion
eduserver.ru
```

You don't have to include `*.` currently.